docs/api/constants.rst

```diff
@@ -730,6 +730,17 @@ Connection
       :py:const:`DRIZZLE_CAPABILITIES_PROTOCOL_41`,
       :py:const:`DRIZZLE_CAPABILITIES_SECURE_CONNECTION`
 
+.. c:type:: drizzle_ssl_state_t
```

Comment:
The commit message doesn't mention anything about adding documentation for the enum. Do you want to add that, or do you want to roll these changes into a separate commit?